Fix ev dispatcher version

yd-dhahn opened the pull request:
added some missing elements of the opentrans specifications

sveneisenschmidt commented:
Hello @yd-dhahn!
First, thank you a lot for your contribution and keeping this library alive.

As you probably saw, and already started to fix, some stuff in composer.json is terrible outdated. Also TravisCI failed to run checks.

Do you mind to have a second look?

yd-dhahn commented:
Hey, good to know this project is still alive, its quite useful for me right now. I changed some of the dependencies so that the test cases now run in travis. I also added some Opentrans stuff that was missing and will continue to do that if i find nodes that are specified but not there. Thanks for your work!

sveneisenschmidt commented:
@yd-dhahn Thanks for updating. Do you mind removing the composer.lock, nowadays it is not needed anymore?

We could also modify the version constraint for symfony/event-dispatcher to support symphony ^3.0 and recent versions of jms/serialiser. What do you think?

yd-dhahn commented:
Hey,
I updated the symfony/event-dispatcherto version 3.4 and the serializer to version 1.11. Most of the things worked quite well, although i had to add the readonly annotation on the @XmlKeyValuePairs for deserialization. Two of the test cases give errors now and I can't really figure out why they are breaking. My guess is that it has to do something with PartyCollectionNode Annotations, maybe you can have a look there?

sveneisenschmidt commented:
@yd-dhahn Thanks, I’ll have a look.
